pkg/certs/util.go

```diff
@@ -68,3 +71,60 @@ func CertificateBundleToString(bundle []*x509.Certificate) string {
 	}
 	return strings.Join(output, "\n")
 }
+
+func ValidatePrivateKey(pemKey []byte) []error {
+	if len(pemKey) == 0 {
+		return []error{fmt.Errorf("required private key is empty")}
+	}
+	if _, err := keyutil.ParsePrivateKeyPEM(pemKey); err != nil {
+		return []error{fmt.Errorf("failed to parse the private key")}
```

Comment:
It might be worthwhile to surface the error returned by the `ParsePrivateKeyPEM` function.

Reply:
Done